npm/ng-packs/packages/schematics/src/utils/text.ts

```diff
@@ -10,6 +10,9 @@ export const macro = (text: string) => upper(snake(text));
 export const dir = (text: string) =>
   strings.dasherize(text.replace(/\./g, '/').replace(/\/\//g, '/'));
 
+// TODO: Is there a standard function to do this? There is a leak if the value contains single quote.
+export const quote = (value: number | string) => typeof value === 'string' ? `'${value}'` : value;
```

Comment:
```suggestion
export const stringify = (value: number | string) => JSON.stringify(value);
```

Reply:
It's awesome. But JSON used double quote symbol `"` rather than single quote symbol `'` which the Angular project prefer.

Reply:
Yeah, you're right.

How about wrapping strings with backticks? I have updated my suggestions accordingly.

Reply:
It doesn't solve the essential. What about if the value contains backtick symbol?
I think single quote is better than backtick. They both have their own leak. But in most of cases, the value won't trigger the leak.

Reply:
I think my origin solution is enough if there is no other standard function to surround a value as code content.

Reply:
Backtick is not as common as a single quote, but OK.

Take a look at my final suggestion. It should solve all cases.